CLAUDE.md

```
@@ -0,0 +1,41 @@
+```bash
+make install          # Install all dependencies
+make start            # Start dev server (after docker compose up redis postgres opensearch)
+pnpm typecheck        # Type check (uses tsgo, fast)
+pnpm test:unit        # Unit tests
+pnpm test:integration # Integration tests
+pnpm test:e2e         # E2E tests
+```
+
+## Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)